src/tools/vmap4_extractor/vmapexport.cpp

```diff
@@ -227,7 +227,7 @@ void ParsMapFiles()
     for (unsigned int i = 0; i < map_count; ++i)
     {
         sprintf(id, "%03u", map_ids[i].id);
-        sprintf(fn, "World\\Maps\\%s\\%s.wdt", map_ids[i].name, map_ids[i].name);
+        sprintf(fn, R"(World\Maps\%s\%s.wdt)", map_ids[i].name, map_ids[i].name);
```

Comment:
To resolve the build error you should swap sprintf with sprintf_s:

sprintf_s(fn, 512, R"(World\Maps\%s\%s.wdt)", map_ids[i].name, map_ids[i].name);

[you can find more information about it on](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/sprintf-s-sprintf-s-l-swprintf-s-swprintf-s-l?view=msvc-160)